cmd/ocsp-responder/main.go

```diff
@@ -306,22 +305,21 @@ func (src *dbSource) Response(ctx context.Context, req *ocsp.Request) ([]byte, h
 		secondaryParsed, err := ocsp.ParseResponse(secondaryResult.bytes, nil)
 		if err != nil {
 			src.log.AuditErrf("parsing secondary OCSP response: %s", err)
-			src.metrics.ocspLookups.WithLabelValues("redis_parse_error").Inc()
-			src.metrics.sourceUsed.WithLabelValues("mysql").Inc()
+			src.metrics.ocspLookups.WithLabelValues("redis", "failed", "other").Inc()
```

Comment:
Same error classifications can be used here as well.

Reply:
Addressed in previous comment thread.